src/content/test/semantics/data-tables.tsx

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { create, React } from '../../common';
+
+export const infoAndExamples = create(({ Markup, Link }) => (
+    <>
+        <h1>Data tables</h1>
+        <p>Semantic elements in a data table must not be coded as presentational.</p>
+
+        <h2>Why it matters</h2>
+        <p>
+            The semantic elements in a data table communicate relationships between cells and row or column headers. When{' '}
+            <Markup.Code>role="presentation"</Markup.Code> is applied to a data table element, its semantics are suppressed, and critical
+            relationships are consequently hidden from people who use assistive technologies. Moreover, they can't use the standard
+            navigational shortcuts for data tables.
+        </p>
+        <p>
+            (Applying <Markup.Code>role="presentation"</Markup.Code> is appropriate for layout tables, because their table semantics do not
+            communicate any meaningful relationships.)
+        </p>
+
+        <h2>How to fix</h2>
+        <p>
+            Do not use <Markup.Code>role="presentation"</Markup.Code> on any of the semantic elements in a data table:
+            <ul>
+                <li>{'<table>'}</li>
+                <li> {'<tr>'}</li>
```

Comment:
There are extra spaces between `<li>` and opening braces `{` here and on following lines - but they do not cause any problems.
```suggestion
                <li>{'<tr>'}</li>
```